<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDatabaseListFormClass</class>
 <widget class="QDialog" name="EditDatabaseListFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditDatabaseListForm</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP/Server</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Port</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>空白行追加</string>
   </property>
   <property name="icon">
    <iconset resource="ButtonSelectDatabase.qrc">
     <normaloff>:/Qtres/Add.png</normaloff>:/Qtres/Add.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>選択行の修正登録</string>
   </property>
   <property name="icon">
    <iconset resource="ButtonSelectDatabase.qrc">
     <normaloff>:/Qtres/Row.png</normaloff>:/Qtres/Row.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>選択行の削除</string>
   </property>
   <property name="icon">
    <iconset resource="ButtonSelectDatabase.qrc">
     <normaloff>:/Qtres/Delete.png</normaloff>:/Qtres/Delete.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>IP/Server</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>File</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
   <property name="icon">
    <iconset resource="ButtonSelectDatabase.qrc">
     <normaloff>:/Qtres/Close.png</normaloff>:/Qtres/Close.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPortNo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>3050</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ButtonSelectDatabase.qrc"/>
 </resources>
 <connections/>
</ui>
